mmv1/products/iamworkforcepool/WorkforcePoolProviderKey.yaml

```diff
@@ -0,0 +1,166 @@
+    type: String
+    description: |
+      The ID of the workforce pool.
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'providerId'
+    type: String
+    description: |
+      The ID of the provider.
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'keyId'
+    type: String
+    description: |
+      The ID to use for the key, which becomes the final component of the resource name. This value must be 4-32 characters, and may contain the characters [a-z0-9-].
+    url_param_only: true
+    required: true
+    immutable: true
+    validation:
+      function: 'ValidateWorkforcePoolProviderKeyId'
+  - name: 'name'
+    type: String
+    description: |
+      Identifier. The resource name of the key.
+      Format: `locations/{location}/workforcePools/{workforcePoolId}/providers/{providerId}/keys/{keyId}`
+    output: true
+  - name: 'keyData'
+    type: 'NestedObject'
+    description: |
+      Immutable. Public half of the asymmetric key.
+    required: true
+    properties:
+      - name: 'format'
+        type: Enum
+        description: |
+          Output only. The format of the key.
+        output: true
+        enum_values:
+          - 'KEY_FORMAT_UNSPECIFIED'
+          - 'RSA_X509_PEM'
+      - name: 'notBeforeTime'
+        type: String
+        description: |
+          Output only. Earliest timestamp when this key is valid. Attempts to use this key before this time will fail.
+          Only present if the key data represents a X.509 certificate.
+
+          Uses RFC 3339, where generated output will always be Z-normalized and uses 0, 3, 6 or 9 fractional digits.
+          Offsets other than "Z" are also accepted.
+          Examples: "2014-10-02T15:01:23Z", "2014-10-02T15:01:23.045123456Z" or "2014-10-02T15:01:23+05:30".
+        output: true
+      - name: 'notAfterTime'
+        type: String
+        description: |
+          Output only. Latest timestamp when this key is valid. Attempts to use this key after this time will fail.
+          Only present if the key data represents a X.509 certificate.
+
+          Uses RFC 3339, where generated output will always be Z-normalized and uses 0, 3, 6 or 9 fractional digits.
+          Offsets other than "Z" are also accepted.
+          Examples: "2014-10-02T15:01:23Z", "2014-10-02T15:01:23.045123456Z" or "2014-10-02T15:01:23+05:30".
+        output: true
+      - name: 'key'
+        type: String
+        description: |
+          Output only. The key data. The format of the key is represented by the format field.
+        output: true
+      - name: 'keySpec'
+        type: Enum
+        description: |
+          Required. The specifications for the key.
+        required: true
+        enum_values:
+          - 'RSA_2048'
+          - 'RSA_3072'
+          - 'RSA_4096'
+  - name: 'state'
+    type: Enum
+    description: |
+      Output only. The state of the key.
+      * STATE_UNSPECIFIED: State unspecified.
```

Comment:
there's also no need to specify the enum values in the description. Generation does this also, see an example in [PubSub.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/a100501b1a8d2a0c27e0fcdf8512490548c34ad3/mmv1/products/pubsub/Topic.yaml#L166-L173) with the generated docs being found [here](https://registry.terraform.io/providers/hashicorp/google/6.34.0/docs/resources/pubsub_topic#encoding-1)